src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -859,12 +862,24 @@ def get_vg(vg_name=None, vg_tags=None, vgs=None):
     return vgs.get(vg_name=vg_name, vg_tags=vg_tags)
 
 
+def get_device_vgs(device, name_prefix=''):
+    stdout, stderr, returncode = process.call(
+        ['pvs', '--noheadings', '--readonly', '--units=g', '--separator=";"',
+         '-o', VG_FIELDS, device],
+        verbose_on_failure=False
+    )
+    vgs = _output_parser(stdout, VG_FIELDS)
+    return [VolumeGroup(**vg) for vg in vgs]
```

Comment:
I think it would be better to return Volumes over just a list, at least for now. Maybe use something like Volumes._populate()?

Reply:
I specifically didn't use Volumes as we don't need the filter feature with this primitive. Is there any advantage to using Volumes?

Reply:
I am aiming for homogeneity. It is confusing to know whether given variable holding a list of LVs is a built-in type list or a class derived from it. Therefore, it would be simpler to keep one.

Reply:
Having an API that defines objects with a set of conventions and then have a separate outlier is going to be surprising. Consistency and a lack of surprises would be my preference. Agreeing with @rishabh-d-dave here

Reply:
Fair points, but this is not the final state. See https://github.com/ceph/ceph/pull/30102#discussion_r344698700